valheim_plus.cfg

```diff
@@ -232,6 +238,22 @@ dataRate=60
 ; The interval in seconds that the game auto saves at (client only)
 autoSaveInterval=1200
 
+[Sleep]
+enabled=false
+; Change false to true to enable this section
+
+byPlayers= false
```

Comment:
I think we should only use `byPercentage` for sleep configuration at least for now. Having these two will create some confusion and I think the percentage setting would be more valuable. What do you think?

Reply:
I think the player is more valuable and only added the percentage based on discussion, and it is always more confusing for players to understand what 33% of 11 players is, but 3 players are always 3 players.

Reply:
That and they are both there for that exact purpose, you don't have to turn both on, you can only use one of them if you care to.